src/App.jsx

```diff
@@ -14,70 +15,63 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const isAdded = (good) => {
+    setSelectedGood(good);
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const isDeleted = () => {
+    setSelectedGood('');
+  };
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood ? (
+        <h1 className="title is-flex is-align-items-center">
+          {selectedGood}
+          {' is selected'}
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={isDeleted}
+          />
+        </h1>
+      ) : (
+        <h1 className="title">No goods selected</h1>
+      )}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const isSelected = selectedGood === good;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+            return (
+              <tr
+                data-cy="Good"
+                className={isSelected ? 'has-background-success-light' : ''}
+                key={good}
+              >
+                <td>
+                  <button
+                    data-cy={isSelected ? 'RemoveButton' : 'AddButton'}
+                    type="button"
+                    className={`button ${isSelected ? 'is-info' : ''}`}
+                    onClick={isSelected ? isDeleted : () => isAdded(good)}
```

Comment:
```suggestion
                    onClick={isSelected ? isDeleted : isAdded(good)}
```
better to modify isAdded function which cant return handler. it will avoid using anonymous arrow function in jsx